easybuild/easyconfigs/l/libxml2/libxml2-2.9.2-goolf-1.5.16.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'ConfigureMake'
+
+name = 'libxml2'
+version = '2.9.2'
+
+homepage = 'http://xmlsoft.org/'
+description = """Libxml2 is the XML C parser and 
+toolchain developed for the Gnome project
+ (but usable outside of the Gnome platform)."""
+
+toolchain = {'name': 'goolf', 'version': '1.5.16'}
+toolchainopts = {'pic': True}
+
+source_urls = [
+    'http://xmlsoft.org/sources/',
+    'http://xmlsoft.org/sources/old/'
+]
+sources = [SOURCELOWER_TAR_GZ]
+
+configopts = 'CC="$CC" CXX="$CXX" --with-pic --without-python --with-zlib=$EBROOTZLIB'
+
+dependencies = [('zlib', '1.2.8')]
+
+sanity_check_paths = {
+    'files': [('lib/libxml2.a', 'lib64/libxml2.a'), ('lib/libxml2.%s' % SHLIB_EXT, 'lib64/libxml2.%s' % SHLIB_EXT)],
+    'dirs': ['bin', 'include/libxml2/libxml'],
+}
+
```

Comment:
@SimonPinches please make sure `include/libxml2` is added to `$CPATH`, cfr. https://github.com/hpcugent/easybuild-easyconfigs/pull/3479/files

This will probably fix this problem I'm seeing when testing your PR:

```
compilation terminated.                                                                                                                                                                                                                                                                                                
make[1]: *** [src/xml_edit.o] Error 1                                                                                                                                                                                                                                                                                  
src/xml_select.c:38:25: fatal error: libxml/tree.h: No such file or directory                                                                                                                                                                                                                                          
 #include <libxml/tree.h>   
```